user/dua-cli/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "dua-cli"
```

Comment:
Since the binary is `dua` and I don't believe there's a need to distinguish between a non-CLI version, perhaps the package should be `dua`? I see packages for other platforms are split on whether to call it `dua` or `dua-cli`.

Reply:
I chose this name because of this part in our docs:

https://github.com/chimera-linux/cports/blob/7fd54f3a46e256c1ef9befdadce009d154606711/Packaging.md?plain=1#L643-L646

The name of the upstream repo is dua-cli, and the name of the crates.io package is also dua-cli.

I'm fine with renaming it though, if you think `dua` is more fitting. I agree, but thought our docs disagreed.